e2e/WORKSPACE

```diff
@@ -15,8 +15,8 @@ http_archive(
 
 http_archive(
```

Comment:
we should probably call the `repositories.bzl` helper from rules_webpack instead of an extra fetch of build_bazel_rules_nodejs

Reply:
I don't think we were fetching twice since, but I agree that we should use `repositories.bzl`.
Done